cirq/google/engine/engine.py

```diff
@@ -206,49 +217,29 @@ def __init__(self,
                 discoveryServiceUrl=self.discovery_url,
                 **service_args)
 
-    def run(
+    def copy(
```

Comment:
Calling `copy` is going to cause another call to the discovery URL service. That seems surprising.